python/mxnet/amp/lists/symbol.py

```diff
@@ -0,0 +1,177 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# coding: utf-8
+"""Lists of functions whitelisted/blacklisted for automatic mixed precision in symbol API."""
+
+FP16_FUNCS = [
+    'Convolution',
+    'Deconvolution',
+    'FullyConnected',
+    'RNN',
+    ]
+
+FP32_FUNCS = [
```

Comment:
Can we add  support in the API for updating the lists or using custom lists for the user ? 
User may want to experiment with certain ops being run in FP16 vs FP32 to see accuracy changes.

Reply:
Makes sense, I will add this today.

Reply:
Done.